sunpy/coordinates/tests/test_utils.py

```diff
@@ -2,14 +2,14 @@
 
 import numpy as np
 
-from astropy.coordinates import SkyCoord
+from astropy.coordinates import SkyCoord, BaseCoordinateFrame, Galactic
```

Comment:
```suggestion
from astropy.coordinates import SkyCoord
```
You don't use `BaseCoordinateFrame` at all in this file.